### PR TITLE
Enhancement/138/spectrum plotter

### LINKED ISSUE #138: Add a spectrum plotter
There might be already one handy

### PR DESCRIPTION
**Pull request by JoschD**
Fixes #138

### CODE REVIEW

File: tests/unit/test_plot_spectrum.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import os
+import pytest
+import numpy as np
+import pandas as pd
+from . import context
+from plot_spectrum import main as plot_spectrum
+
+
+import tempfile
+
```
Comment (mihofer) [inline tests/unit/test_plot_spectrum.py]: maybe add test to check if return values are ok?

Comment (JoschD) [reply tests/unit/test_plot_spectrum.py]: added. Noticed also possible problem with non-unique filenames. Solved in new commit

File: omc3/plot_spectrum.py [hunk 2]
```diff
@@ -0,0 +1,751 @@
...
+
+LOG = logging_tools.getLogger(__name__)
+
+PLANES = ('x', 'y')
+
+STEM_LINES_ALPHA = 0.5
+RESONANCE_LINES_ALPHA = 0.5
+
+LABEL_Y_SPECTRUM = 'Amplitude in {plane:s} [a.u]'
+LABEL_Y_WATERFALL = 'Items in {plane:s}'
```
[the remark this answers is not in the document]
Comment (JoschD) [reply omc3/plot_spectrum.py]: generic

**Comment by JoschD** [comment]
Decided it still needs option to create single plot from multiple files.  Maybe via --data_mode option

**Pull request merged**